app/Controllers/feedController.php

```diff
@@ -734,19 +734,14 @@ public function reloadAction() {
 
 		//Extract all feed entries from database, load complete content and store them back in database.
 		$entries = $entryDAO->listWhere('f', $feed_id, FreshRSS_Entry::STATE_ALL, 'DESC', 0);
-
-		//We need another DB connection in parallel
-		Minz_ModelPdo::$usesSharedPdo = false;
-		$entryDAO2 = FreshRSS_Factory::createEntryDao();
+		$entries = iterator_to_array($entries);	//TODO: Improve
```

Comment:
For debugging: This is to disable streaming = load all articles in memory. But we might exhaust memory by doing so.

Reply:
The tested feed retrieves only 10 articles. We're good for the memory consumption :)

Reply:
I think we need to introduce an explicit limit there. The line a bit higher up retrieves all articles for that feed, which can be several thousands...